policy/store.go

```diff
@@ -26,6 +26,33 @@ func StoreFromDataFiles(paths []string) (storage.Store, error) {
 	return store, nil
 }
 
+// ReadDataFiles returns the paths to all data documents
+// at the given path. Any JSON or YAML document
+// could be a valid document.
+func ReadDataFiles(path string) ([]string, error) {
+	var data []string
+	err := filepath.Walk(path, func(currentPath string, info os.FileInfo, err error) error {
+		if err != nil {
+			return fmt.Errorf("walk path: %w", err)
+		}
+
+		if info.IsDir() {
+			return nil
+		}
+
+		if filepath.Ext(currentPath) == ".json" || filepath.Ext(currentPath) == ".yaml" {
```

Comment:
nit: `.yml` would also be valuable to catch

Reply:
Nice catch! I always forget `.yml` is a thing